sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 12.13.0-beta.1 (Unreleased)
```

Comment:
TODO: merge changelog update PR, then merge main. This will present a merge conflict if we merge this PR before the other one (i think).

Reply:
Changelog PR has merged.